airflow_two/Dockerfile

```diff
@@ -91,6 +91,7 @@ RUN set -ex \
     && pip install psycopg2 \
     && pip install google-cloud-logging==1.11.0 \
     && pip install retrying==1.3.3 \
+    && pip install attrs==23.2.0 \
```

Comment:
This version is already used e.g. here https://github.com/medicode/diseaseTools/blob/66ce20fdf712d0e53763417f7fa4639a98ff32c9/docker/common/shared_requirements.txt#L31 and in multiple other images.